src/lib/KeyStore.js

```diff
@@ -185,6 +191,7 @@ class KeyStore {
                 id: legacyKeyId,
                 type: Key.Type.LEGACY,
                 encrypted: true,
+                hasPin: key.type === 'low',
```

Comment:
How I get it `key.type` is a `Key.Type` (which is `0` or `1`) so it should never be `'low'` here. Is this legacy code?